src/tests/ftest/util/ec_utils.py

```diff
@@ -503,14 +501,20 @@ def write_single_fio_dataset(self, results):
             self.execute_fio(stop_dfuse=False)
             if results is not None:
                 results.put("PASS")
-        except (CommandFailure, DaosApiError, DaosTestError):
+        except (CommandFailure, DaosApiError, DaosTestError) as error:
             if results is not None:
-                results.put("FAIL")
+                results.put("FAIL - {}".format(error))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string